graphrag/config/init_content.py

```diff
@@ -77,7 +77,7 @@
 vector_store:
   type: {vector_store_defaults.type}
   db_uri: {vector_store_defaults.db_uri}
-  container_name: {vector_store_defaults.container_name}
+  index_prefix: {vector_store_defaults.index_prefix}
```

Comment:
Can you remove this from `init_content` completely? I'd like it to be a lesser-used option, so keeping it out of the default settings.yaml will make it less obvious.